src/schema/credit_card.js

```diff
@@ -16,6 +20,15 @@ const CreditCardMutationSuccessType = new GraphQLObjectType({
       type: CreditCard.type,
       resolve: creditCard => creditCard,
     },
+    creditCardEdge: {
+      type: CreditCardEdge,
+      resolve: creditCard => {
+        return {
+          cursor: cursorForObjectInConnection([creditCard], creditCard),
```

Comment:
😍

Reply:
Good question (and actually question for you @alloy)-- it's true that initially there will be only one CC. But-- reading [here](https://facebook.github.io/relay/graphql/connections.htm#sec-Forward-pagination-arguments) it seems that you _need_ a `cursor` field, but the cursor value only matters if you're using relay-style pagination.

Our UI doesn't paginate, and it would be an extra call to fetch the entire array of your credit cards. It feels like I'm missing something, but is it acceptable to just do what I'm doing here? Should I add a comment? Or is there another way?

Reply:
Great question as well! From the cursor-pagination spec it’s not entirely clear to me _why_ this is needed, nor do I know if Relay Modern actually still makes use of it, so I’ll have to dive into this a bit (I actually have a TODO for some work on these specs anyways, which is to remove the use of the word ‘Relay’). I’ll get back to you on that, for now this is definitely a fine solution 👍